v3.2/glfw/build.go

```diff
@@ -33,11 +33,13 @@ package glfw
 // FreeBSD Build Tags
 // ----------------
 // GLFW Options:
-#cgo freebsd,!wayland CFLAGS: -D_GLFW_X11 -D_GLFW_HAS_GLXGETPROCADDRESSARB -D_GLFW_HAS_DLOPEN
-#cgo freebsd,wayland CFLAGS: -D_GLFW_WAYLAND -D_GLFW_HAS_DLOPEN
+#cgo freebsd pkg-config: glfw3
+#cgo freebsd CFLAGS: -D_GLFW_HAS_DLOPEN
+#cgo freebsd,!wayland CFLAGS: -D_GLFW_X11 -D_GLFW_HAS_GLXGETPROCADDRESSARB
+#cgo freebsd,wayland CFLAGS: -D_GLFW_WAYLAND
 
 // Linker Options:
-#cgo freebsd,!wayland LDFLAGS: -lGL -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm -lXinerama
-#cgo freebsd,wayland LDFLAGS: -lGL -lwayland-client -lwayland-cursor -lwayland-egl -lxkbcommon -lm
+#cgo freebsd,!wayland LDFLAGS: -lm -lGL -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lXinerama
+#cgo freebsd,wayland LDFLAGS: -lm -lGL -lwayland-client -lwayland-cursor -lwayland-egl -lxkbcommon
```

Comment:
I was wondering why you changed the order here.

Reply:
I was testing out a few different variants of this change to see what was actually required here. I kept it mostly as-is but made it clearer which libs are the same across both these targets.

Reply:
I'm not sure (I became a member this month) but there might be an intention to keep this in ASCII order.